### PR TITLE
tikv benchmark overview

### PR DESCRIPTION
**Pull request by iosmanthus**
Signed-off-by: iosmanthus <myosmanthustree@gmail.com>

add an overview to TiKV YCSB benchmark result.

### Which version does your change affect?
v5.0.2

### CODE REVIEW

**Comment by iosmanthus** [comment]
@marsishandsome PTAL

File: content/docs/5.1/deploy/ performance/overview.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Overview
+description: How to make a benchmark over a TiKV cluster
+menu:
+    "5.1":
+        parent: Performance
+        weight: 5
+---
+
+TiKV delivers predictable throughput and latency at all scales on commodity hardware. This document provides an overview of the performance profiles you can expect, based on PingCAP's testing using industry-standard benchmarks [YCSB](https://github.com/brianfrankcooper/YCSB)
```
Comment (zz-jason) [inline content/docs/5.1/deploy/ performance/overview.md:10]: TiKV requires NVMe disks, maybe it's not suitable to say `commodity hardware`?

File: content/docs/5.1/deploy/ performance/overview.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+For instructions to reproduce the TPC-C results listed here, see [Benchmark Instructions](./instructions.md). If you fail to achieve similar results, there is likely a problem in either the hardware, workload, or test design.
+
+## Baseline
+
+The goal of the YCSB project is to develop a framework and common set of workloads for evaluating the performance of different "key-value" and "cloud" serving stores. **A 3-node TiKV cluster could achieve at most 200,000 OPS and <10ms latency in a 10M records and 10M operations YCSB workload.**
```
Comment (zz-jason) [inline content/docs/5.1/deploy/ performance/overview.md:16]:
```suggestion
The goal of the YCSB project is to develop a framework and common set of workloads for evaluating the performance of different key-value and cloud serving stores. **A 3-node TiKV cluster could achieve at most 200,000 OPS within 10ms latency in a 10M records and 10M operations YCSB workload.**
```

File: content/docs/5.1/deploy/ performance/overview.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+For a refresher on what exactly YCSB is and how it is measured, see [the official instructions to the core YCSB workload](https://github.com/brianfrankcooper/YCSB/wiki/Core-Workloads)
```
Comment (zz-jason) [inline content/docs/5.1/deploy/ performance/overview.md:18]:
```suggestion
For a refresher on what exactly YCSB is and how it is measured, see the official introduction of YCSB [Core Workload](https://github.com/brianfrankcooper/YCSB/wiki/Core-Workloads)
```

File: content/docs/5.1/deploy/ performance/overview.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+TiKV achieves this performance in [linearizability](https://en.wikipedia.org/wiki/Linearizability), a strong correctness condition, which constrains what outputs are possible when an object is accessed by multiple processes concurrently.
+
+We will give a demonstration to the benchmark result of a 3-node TiKV cluster with different client concurrency to give a big picture of the throughput and latency of TiKV.
```
Comment (zz-jason) [inline content/docs/5.1/deploy/ performance/overview.md:22]:
```suggestion
A demonstration of the benchmark result on a 3-node TiKV cluster with different client concurrency will be given to show a big picture of the throughput and latency of TiKV.
```

File: content/docs/5.1/deploy/ performance/overview.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+# Cluster Configuration
+
+We deploy a 3-node cluster in our private cloud environment with the following node configuration.
```
Comment (zz-jason) [inline content/docs/5.1/deploy/ performance/overview.md:26]:
```suggestion
Deploy a 3-node cluster with the following node configuration:
```

Comment (zz-jason) [reply content/docs/5.1/deploy/ performance/overview.md:26]: don't use `we`, `you`, `I`, `me`, please use objective things.

File: content/docs/5.1/deploy/ performance/overview.md (hunk 1 above)
Comment (iosmanthus) [reply content/docs/5.1/deploy/ performance/overview.md:10]: We may use `required` or `requiring`?

File: content/docs/5.1/deploy/ performance/overview.md (hunk 5 above)
Comment (iosmanthus) [reply content/docs/5.1/deploy/ performance/overview.md:26]: Noted.

**Comment by iosmanthus** [comment]
@zz-jason PTAL

File: content/docs/5.1/deploy/ performance/performance.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+title: Performance
```
Comment (marsishandsome) [inline content/docs/5.1/deploy/ performance/performance.md:2]:
```suggestion
title: Benchmark and Performance
```

**Review by marsishandsome — COMMENTED** [review]
there is an extra space before the folder name `performance`

File: content/docs/5.1/deploy/ performance/overview.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
+---
+title: Overview
+description: How to make a benchmark over a TiKV cluster
+menu:
+    "5.1":
+        parent: Performance
+        weight: 5
+---
+
+TiKV delivers predictable throughput and latency at all scales on requiring hardware. This document provides an overview of the performance profiles you can expect, based on PingCAP's testing using industry-standard benchmarks [YCSB](https://github.com/brianfrankcooper/YCSB)
```
Comment (marsishandsome) [inline content/docs/5.1/deploy/ performance/overview.md:10]:
```suggestion
TiKV delivers predictable throughput and latency at all scales on requiring hardware. This document provides an overview of the performance profiles you can expect, based on PingCAP's testing using industry-standard benchmarks [YCSB](https://github.com/brianfrankcooper/YCSB)'s go version [GO YCSB](https://github.com/pingcap/go-ycsb).
```

File: content/docs/5.1/deploy/ performance/overview.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+## Latency
+
+TiKV fits to latency-sensitive service, we could achieve <10ms average latency even in a high pressure throughput. You can check the average latency in [Figure 2](https://docs.google.com/spreadsheets/d/e/2PACX-1vTIx695jjL3qYN1iR4xC3N8qh0B1qsHOALSBqf1B469b0DIZwVdzZMcSbBOOtAIo31hAdW0x_EXjmgq/pubchart?oid=334435174&format=interactive).
+
+{{< figure
+    src="/img/docs/avg-latency.svg"
+    caption="YCSB latency"
+    width="1000"
+    number="1" >}}
```
Comment (marsishandsome) [inline content/docs/5.1/deploy/ performance/overview.md:53]:
```suggestion
    number="2" >}}
```

File: content/docs/5.1/deploy/ performance/overview.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+For the 99th percentile latency, see [Figure 3](https://docs.google.com/spreadsheets/d/e/2PACX-1vTIx695jjL3qYN1iR4xC3N8qh0B1qsHOALSBqf1B469b0DIZwVdzZMcSbBOOtAIo31hAdW0x_EXjmgq/pubchart?oid=6574505&format=interactive)
+
+{{< figure
+    src="/img/docs/99-latency.svg"
+    caption="YCSB 99th percentile latency"
+    width="1000"
+    number="1" >}}
```
Comment (marsishandsome) [inline content/docs/5.1/deploy/ performance/overview.md:61]:
```suggestion
    number="3" >}}
```

Comment (iosmanthus) [reply content/docs/5.1/deploy/ performance/overview.md:61]: Done.

File: content/docs/5.1/deploy/ performance/overview.md (hunk 8 above)
Comment (iosmanthus) [reply content/docs/5.1/deploy/ performance/overview.md:53]: Done.

File: content/docs/5.1/deploy/ performance/overview.md (hunk 7 above)
Comment (iosmanthus) [reply content/docs/5.1/deploy/ performance/overview.md:10]: Done.

File: content/docs/5.1/deploy/ performance/performance.md (hunk 6 above)
Comment (iosmanthus) [reply content/docs/5.1/deploy/ performance/performance.md:2]: Done.

**Comment by iosmanthus** [comment]
@marsishandsome PTAL

**Review by marsishandsome — APPROVED** [review]
LGTM

**Comment by iosmanthus** [comment]
@zz-jason PTAL

**Review by zz-jason — APPROVED** [review]
LGTM

**Pull request merged**